chaos_genius/core/utils/kpi_validation.py

```diff
@@ -8,52 +8,62 @@
 
 from chaos_genius.core.rca.root_cause_analysis import SUPPORTED_AGGREGATIONS
 from chaos_genius.core.utils.data_loader import DataLoader
+from chaos_genius.databases.models.data_source_model import DataSource
 from chaos_genius.settings import MAX_ROWS_FOR_DEEPDRILLS
 
 KPI_VALIDATION_TAIL_SIZE = 1000
 
 logger = logging.getLogger(__name__)
 
 
-def validate_kpi(kpi_info: Dict[str, Any], data_source: Dict[str, Any]) -> Tuple[bool, str]:
+def validate_kpi(kpi_info: Dict[str, Any], check_tz_aware: bool = False) -> Tuple[bool, str, bool]:
```

Comment:
The last return value should be an `Optional[bool]` if it can be None

Reply:
Done.